### ISSUE TITLE
crx and xpi files in releases

### ISSUE THREAD

**Issue by gtitov** [joelshepherd/tabliss#422]
Hello!
Tabliss is a great product and I really enjoy using it.

# Description

I had difficulties installing Tabliss in Ungoogled Chromium. There is no "Add to browser" button in Chrome Web Store when you use Chromium and it is now impossible to download .crx file from Chrome Web Store.

I found a workaround with CRX Extractor/Downloader extension in Google Chrome but I definitely would like to download it directly from the developers.

# Request

Would you consider providing .crx and .xpi files in releases or on the Tabliss website?

Of course it is possible to build these files from source but I would rather download ready-to-use files.

**Comment by joelshepherd** [comment]
Yeah fair, I am planning to rewrite the CI pipeline to github actions. I'll add this when I do the rewrite

**Comment by joelshepherd** [comment]
Hi @gtitov, do you use `crx` or `zip` files when installing your extensions manually?

I've trialed building `crx`s, but I cannot import these into my chromium (standard chromium without google api keys, not the ungoogled version specifically) because it is not uploaded to the chrome web store/signed by a google certificate. But I can extract an extension zip and use "Load Unpacked" no worries.

So what I have done so far is add a `zip` of the build to the CI pipeline (note, these are nightly builds): https://github.com/joelshepherd/tabliss/actions/runs/1937098061

If these zips work for you, I will look into setup releases and attaching these zips there.

**Comment by gtitov** [comment]
Hi!

Zips work just fine. Thank you! I've unzipped and successfully installed `tabliss-chromium.zip` on Chromium Version 95.0.4638.69 (Official Build, ungoogled-chromium) (64-bit).
